packages/Webkul/Shop/src/Http/Controllers/OrderController.php

```diff
@@ -70,7 +83,16 @@ public function view($id)
             abort(404);
         }
 
-        return view($this->_config['view'], compact('order'));
+        foreach ($order->items as $item) {
+            $slugOrPath = strtolower(str_replace(" ","-",$item->name));
+            $product = $this->productRepository->findBySlug($slugOrPath);
```

Comment:
You can directly get product from $item. (Example $item->product)

Reply:
Thanks for your tip it help me to reduce no of line in my code.